files/export-service-util.py

```diff
@@ -0,0 +1,200 @@
+import json
+import argparse
+import tempfile
+from azureml.core import Workspace
+from azureml.exceptions import WebserviceException
+from azureml.core.compute import KubernetesCompute, ComputeTarget
+from azureml._model_management._util import _get_mms_url
+from azureml._model_management._util import get_paginated_results
+from azureml._model_management._util import get_requests_session
+from azureml._model_management._constants import AKS_WEBSERVICE_TYPE, ACI_WEBSERVICE_TYPE, UNKNOWN_WEBSERVICE_TYPE
+from azureml._model_management._constants import MMS_SYNC_TIMEOUT_SECONDS
+from azureml.core.webservice import Webservice
+from azureml._restclient.clientbase import ClientBase
+
+MIGRATION_WEBSERVICE_TYPES = [AKS_WEBSERVICE_TYPE, ACI_WEBSERVICE_TYPE]
+
+
+def attach(ws: Workspace,
+           compute_name: str,
+           cluster_name: str,
+           namespace: str = None):
+    """
+    Attach AmlArc compute to the workspace.
+    :param cluster_name: AKS cluster name.
+    :param namespace: Compute namespace, if not provided will be set to 'default' namespace
+    :param compute_name: Compute name.
+    :param ws: Target workspace.
+    """
+    resource_id = f"'/subscriptions/{ws.subscription_id}/resourceGroups/{ws.resource_group}/providers/" \
+                  f"Microsoft.ContainerService/managedClusters/{cluster_name}"
+    compute_id = None
+    if compute_name in ws.compute_targets:
+        amlarc_compute = ws.compute_targets[compute_name]
+        if amlarc_compute and type(amlarc_compute) is KubernetesCompute:
+            compute_id = amlarc_compute.id
+    else:
+        amlarc_attach_configuration = KubernetesCompute.attach_configuration(resource_id, namespace=namespace)
+        amlarc_compute = ComputeTarget.attach(ws, compute_name, amlarc_attach_configuration)
+        amlarc_compute.wait_for_completion(show_output=True)
+        compute_id = amlarc_compute.id
+    print(compute_id)
+
+
+def export(ws: Workspace,
+           compute_type: str = None,
+           timeout_seconds: int = None,
+           show_output: bool = True) -> (str, str):
+    """
+    Export all services under target workspace into template and parameters,
+    valid compute_types are "AKS"/"ACI", default to export for both two types.
```

Comment:
compute type needs align with line 15

Reply:
They have aligned with constants, for readable message only.